test/integration/suites/nested-rotation/Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM spire-agent:latest-local AS nested-agent
-RUN apk add --update openssl && \
-    rm -rf /var/cache/apk/*
 CMD []
+RUN apk add --no-cache --update openssl && \
+    rm -rf /var/cache/apk/*
```

Comment:
Since this has been updated to use `--no-cache`, should we drop the `rm -rf` invocation afterwards?